.github/workflows/build-and-test.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-          python-version: [3.8]
+          python-version: [3.8, "3.10"]
```

Comment:
@nikolas just noting this number is in quotations. Intentional? Just asking since I don't remember seeing this yet

Reply:
Yeah, this is intentional because in python the number `3.10` will evaluate to `3.1`, which is a whole different python version. Apparently the root issue here is known as "The Norway Problem": https://dev.to/hugovk/the-python-3-1-problem-85g

https://travis-ci.community/t/add-python-3-10/12220/2

Reply:
Ah! Got it